src/components/tabs/index.tsx

```diff
@@ -4,13 +4,15 @@ import { Tab, TabButtonControls, TabDropdownControls } from './components'
 import { useOverflowRef, useTabItems, useSyncedTabGroups } from './hooks'
 import { TabItem, TabsProps } from './types'
 import s from './tabs.module.css'
+import TabNestingProvider, { useIsNested } from './helpers/tab-nesting-context'
```

Comment:
🎨 Prefer to see the style import last.

```suggestion
import TabNestingProvider, { useIsNested } from './helpers/tab-nesting-context'
import s from './tabs.module.css'
```